drivers/serial/uart_rpi_pico.c

```diff
@@ -90,6 +90,78 @@ static int uart_rpi_init(const struct device *dev)
 	return 0;
 }
 
+static int uart_rpi_config_get(const struct device *dev, struct uart_config *cfg)
+{
+	struct uart_rpi_data *data = dev->data;
+
+	*cfg = data->uart_config;
+	return 0;
+}
+
+static int uart_rpi_configure(const struct device *dev, const struct uart_config *cfg)
+{
+	const struct uart_rpi_config *config = dev->config;
+	uart_inst_t * const uart_inst = config->uart_dev;
+	struct uart_rpi_data *data = dev->data;
+
+	uint data_bits = 0;
```

Comment:
Does this need to be a 32-bit variable?

Reply:
The function declaration is
```
uart_set_format(uart_inst_t *uart, uint data_bits, uint stop_bits, uart_parity_t parity)
```
So the types are correct